board/drivers/timers.h

```diff
@@ -5,14 +5,18 @@ void timer_init(TIM_TypeDef *TIM, int psc) {
   TIM->SR = 0;
 }
 
-void microsecond_timer_init(void) {
-  MICROSECOND_TIMER->PSC = (APB1_FREQ)-1U;
-  MICROSECOND_TIMER->CR1 = TIM_CR1_CEN;
-  MICROSECOND_TIMER->EGR = TIM_EGR_UG;
+void us_timer_init(void) {
```

Comment:
should we make a timer struct instead?

Reply:
ok, will do but in the next separate PR as there is too many places in the safety code